distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestApplicationService.java

```diff
@@ -81,7 +81,7 @@
 
     private static final String CATALOG_APP = "catalog-app";
 
-    private static final String SOLR_APP = "solr-app";
+    private static final String REGISTRY_APP = "registry-app";
```

Comment:
Might be worth making this a comment as well, in case registry-app ever gets started by default.